ci-operator/step-registry/ipi/conf/vsphere/windows-machineset/ipi-conf-vsphere-windows-machineset-ref.yaml

```diff
@@ -0,0 +1,23 @@
+ref:
+  as: ipi-conf-vsphere-windows-machineset
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
+  commands: ipi-conf-vsphere-windows-machineset-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: WINDOWS_USER_DATA_SECRET
+      default: "windows-user-data"
+    - name: WINDOWS_NODE_REPLICAS
+      default: "1"
+    - name: WINDOWS_OS_ID
+      default: "projects/windows-vsphere/images/family/windows-2022-core" 
```

Comment:
This image belongs to GCP. For vSphere you need to identify the vsphere template (golden image) used for CI. Maybe you can try using the one we use for our testing: openshift-qe-winserver-2022-no-containers  https://gitlab.cee.redhat.com/aosqe/flexy-templates/-/blob/master/functionality-testing/aos-4_12/hosts/windows_workers/scaleup_windows_workers.sh#L278

Reply:
Developers seem to be using  vmTemplate = "windows-golden-images/windows-server-2022-template-ipv6-disabled" in their CI, so you could set the template to the same value (as it seems we need to disable IPv6 due to some IBM infra issue)

Reply:
got it .. will make the update